packages/@o3r/testing/schematics/ng-add/templates/workspace/jest.config.ut.js.template

```diff
@@ -14,11 +14,12 @@ globalThis.ngJest = {
  * @returns {import('ts-jest/dist/types').JestConfigWithTsJest}
  */
 module.exports.getJestProjectConfig = (rootDir, isAngularSetup) => {
-  const relativePath = relative(rootDir, '.');
-  const moduleNameMapper = Object.entries(pathsToModuleNameMapper(compilerOptions.paths)).reduce((acc, [moduleName, path]) => {
+  const relativePath = relative(rootDir, __dirname);
+  const moduleNameMapper = Object.entries(pathsToModuleNameMapper(compilerOptions.paths || {})).reduce((acc, [moduleName, path]) => {
```

Comment:
This is also handling the deep imports. No need for extra mappers like `moduleNameMapper['^@o3r/testing/core/(.*)'] = [join(require.resolve('@o3r/testing/core/angular'), '$1')];`